packages/dara-components/js/graphs/graph-layout.tsx

```diff
@@ -9,10 +9,18 @@ import {
     SpringLayout,
     TieredGraphLayoutBuilder,
 } from '@darajs/ui-causal-graph-editor';
+import { GraphLayoutBuilder } from '@darajs/ui-causal-graph-editor/dist/shared/graph-layout/common';
 
 // Types mirror backend types defined in dara.components.graphs.graph_layout
 type GraphLayoutType = 'marketing' | 'planar' | 'spring' | 'circular' | 'fcose' | 'force_atlas' | 'custom';
 
+interface DefinitionWithTiers extends TieredGraphLayoutBuilder {
+    tier_separation?: number;
+}
+
+interface BuilderWithTiers extends TieredGraphLayoutBuilder {
+    tierSeparation?: (separation: number) => GraphLayoutBuilder<unknown>;
+}
```

Comment:
This could be defined next to the builders and extended directly by the tier-supporting builders